and-bible/app/src/main/res/values/strings.xml

```diff
@@ -268,6 +268,77 @@ TODO: error and download_document_confirm_prefix need to be parameterized rather
     <string name="speak_queue">Queue</string>
     <string name="speak_repeat">Repeat</string>
 
+    <!-- Speak - Bible -->
+    <string name="speak_speed_title">Speak speed</string>
+    <string name="conf_speak_synchronize">Synchronize text with speech</string>
+
+    <string name="conf_speak_auto_bookmark">Automatic bookmarking</string>
+    <string name="restore_settings_from_bookmarks">Save playback settings to bookmarks</string>
+
+    <string name="conf_speak">Speak and play earcon sounds for</string>
+    <string name="conf_replace_divinename">Replace divine name with Yahweh</string>
+
+    <string name="conf_change_chapter">Chapter changes</string>
+    <string name="conf_change_title">Titles</string>
+    <string name="conf_speak_footnotes">Footnotes</string>
+
+    <string name="conf_sleep_timer">Set sleep timer</string>
+    <string name="sleep_timer_title">Sleep time in minutes</string>
+    <string name="sleep_timer_timer_set">Set sleep timer. Timer set: %d minutes</string>
+
+    <string name="conf_playback_settings">Playback settings</string>
+    <string name="conf_speak_settings">General speak settings</string>
+    <string name="conf_speak_bookmarking_settings">Bookmarking settings</string>
+    <string name="sleep_timer_started">Sleep timer started, %d minutes</string>
+
+    <string name="speak_next">Next</string>
+    <string name="speak_previous">Previous</string>
+    <string name="widget_toggle_sleep_timer">Toggle sleep timer</string>
+    <string name="bookmarkwidget_helptext">Speak bookmarks shortcut buttons will appear here</string>
+    <string name="sleep_timer_active_at">Sleep timer: %s minutes left.</string>
+    <string name="speak_status_stopped">- stopped - </string>
+    <string name="speak_bookmark_help">Speak bookmark help</string>
+    <string name="speak_bookmark_help_message"><b>Automatic bookmarking</b>
```

Comment:
I need to look at this more closely because it is hard to change text once it has been translated into multiple languages but I think you will be able to simplify this when the verse specific settings are removed.

Reply:
We need to go through all the new strings once more. 
- [ ] I'll go through them first and 
- [ ] then ask for reviewing them.